packages/client/src/objectSet/ObjectSetListenerWebsocket.test.ts

```diff
@@ -419,7 +419,7 @@ function createMockWebSocketConstructor(
         eventEmitter.removeEventListener.bind(eventEmitter),
       ) as any,
 
-      send: vi.fn((a, _b) => {
+      send: vi.fn((a, _b: any) => {
```

Comment:
Why'd you need to add a type for _b?

Reply:
in the old way of the type being args and return value, there was only one way to resolve the function. But now that its `typeof foo` and there  can be overloads, it can't infer properly what this value is because in one case its missing and in another it was defined. This was fine before, we didnt use it and i just told it about the defined value in the type but we don't have that any more.